terraform/outputs.tf

```diff
@@ -13,58 +13,71 @@
 # limitations under the License.
 
 output "gclb_external_ip_name" {
-  description = "The external IPv4 name assigned to the global fowarding rule for the global load balancer."
+  description = "The external IPv4 name assigned to the global fowarding rule for the global load balancer fronting the webhook."
   value       = module.gclb.external_ip_name
 }
 
 output "gclb_external_ip_address" {
-  description = "The external IPv4 assigned to the global fowarding rule for the global load balancer."
+  description = "The external IPv4 assigned to the global fowarding rule for the global load balancer fronting the webhook."
   value       = module.gclb.external_ip_address
 }
 
 output "run_service_url" {
-  description = "The Cloud Run service url."
-  value       = module.cloud_run.url
+  description = "The Cloud Run webhook service url."
+  value       = module.webhook_cloud_run.url
 }
 
 output "run_service_id" {
-  description = "The Cloud Run service id."
-  value       = module.cloud_run.service_id
+  description = "The Cloud Run webhook service id."
+  value       = module.webhook_cloud_run.service_id
 }
 
 output "run_service_name" {
-  description = "The Cloud Run service name."
-  value       = module.cloud_run.service_name
+  description = "The Cloud Run webhook service name."
+  value       = module.webhook_cloud_run.service_name
 }
 
 output "run_service_account_name" {
   description = "Cloud Run service account name."
-  value       = google_service_account.run_service_account.name
+  value       = google_service_account.webhook_run_service_account.name
 }
 
 output "run_service_account_email" {
   description = "Cloud Run service account email."
-  value       = google_service_account.run_service_account.email
+  value       = google_service_account.webhook_run_service_account.email
 }
 
 output "run_service_account_member" {
   description = "Cloud Run service account email iam string."
-  value       = google_service_account.run_service_account.member
+  value       = google_service_account.webhook_run_service_account.member
```

Comment:
You could probably group these into an object and provide an additional object for the retry service, or forego the retry service but splitting it like this allows you to add the retry output later if needed without breaking the output API.